src/components/LinkEditor.tsx

```diff
@@ -140,17 +140,19 @@ class LinkEditor extends React.Component<Props, State> {
       }
 
       case "ArrowUp": {
+        if (event.shiftKey) return;
         event.preventDefault();
         event.stopPropagation();
         const prevIndex = this.state.selectedIndex - 1;
 
         this.setState({
-          selectedIndex: Math.max(0, prevIndex),
+          selectedIndex: Math.max(-1, prevIndex),
```

Comment:
Why change this?

Reply:
it changes the key semantics of the search box to be similar to google (as well as many other search / type-ahead interfaces), where pressing up at the top item puts you back into the editor.

The world is about 50-50 on this behavior but the products that think about UI more intently generally do it this way IMO. It feels right to me